core/scss/components/skiplinks/_skiplinks.scss

```diff
@@ -0,0 +1,31 @@
+@charset "UTF-8";
+
+//
+// Skiplinks
+//
+// For providing a link at the top of the page which jumps
+// the user down to an anchor or target at the beginning of the main content.
+//
+// Gives screen readers and keyboard users the same capability of going directly
+// to the main content as sighted, mouse users.
+// of the main content. Learn more at <a href="https://webaim.org/techniques/skipnav">https://webaim.org/techniques/skipnav</a>.
+//
+// To test this component's functionality, place your cursor on "Example" and click.
+// Then, press the tab key. The Skiplinks link will slide into view at the top
+// lefthand corner of the screen. Press the return key (or click on the link) to
+// skip to the targeted element.
+//
+// Markup: <header>
+//   <a href="#main-content" class="su-skiplinks">Skip to main content</a>
+// </header>
+//  <main>
+//   <h2 id="main-content" class="su-sr-only-text" tabindex="-1">Main Content</h2>
+// </main>
```

Comment:
@sherakama Hardcoding the markup here means the styleguide doesn't use our twig templates. In the other branch I refactored the twig files specifically so that the styleguide could use the roll-up `skiplinks.twig` file, which evaluates to essentially this hardcoded markup, while coding the individual elements separately (`link.twig` & `target.twig`) files so site builders could place the elements appropriately in their sites' markup. So is there still a reason to hardcode the markup here? I think having the styleguide use `skiplinks.twig`, which in turn uses the twig files we're recommending site builders use, is preferable.

Reply:
I was trying to find a way to balance effort, documentation, and keeping the twig templates directly usable vs an example. 

* This was a fast (lazy) way to create the correct output in the styleguide
* Maintained a usable twig file. 

I'd prefer to avoid having twig files merely to generate styleguide output as we can do that in the markup and to keep the intent of the twig files as something that an author can/should/could use directly in their project.

Reply:
The style guide is how I test the twig files. If they're not actually used in the style guide, how do I know they work? I don't want to have to wait until I build a site on top of decanter to find out there's a problem with the templates.